components/avatar.tsx

```diff
@@ -71,6 +71,9 @@ function Avatar({
       {...props}
       onClick={changeable ? handleClick : props.onClick}
       onChange={changeable ? handleFileChange : onChange}
+      data-testid="avatar-content"
```

Comment:
let's try avoiding using `data-testid` as much as possible? perhaps, we can make this element to receive `id` props, and then set the `id` and then we can find the element through that ID.

indeed, setting the ID manually is not possible at all time, but we should be able to find a way, I will discuss below for those cases.